### PR TITLE
Use Strings for Rule Values

### LINKED ISSUE #13: Create rules in plain english
http://eslint.org/docs/user-guide/configuring#configuring-rules

### PR DESCRIPTION
**Pull request by kylecesmat**
This PR converts all rules that are defined with ints to strings. The changes are applied to `/rules` as well as `/configurations`. This is the recommended style and fixes any inconsistencies in rule-usage. Non-breaking.

closes #13

`"0" -> "off"`

`"1" -> "warn"`

`"2" -> "error"`

cc @baer @ryan-roemer

### CODE REVIEW

**Comment by baer** [comment]
Nice work!

**Comment by divmain** [comment]
:+1:
